inst/rmarkdown/templates/visc_report/skeleton/skeleton.Rmd

```diff
@@ -101,14 +101,25 @@ pandoc_markup <- set_pandoc_markup(output_type)
 remotes::install_github("https://github.com/FredHutch/VDCNNNAnalysis.git")
 ```
 
+```{r check_data_version}
+DataPackageR::assert_data_version("VDCNNNAnalysis",version_string = "data_version")
+testthat::expect_equal(packageDescription("VDCNNNAnalysis")$RemoteSha,"commit_sha")
+testthat::expect_equal(packageDescription("VDCNNNAnalysis")$RemoteRef,"branch_name")
+
```

Comment:
Editing this further... shouldn't this be for the datapackage not for the analysis folder (since the analysis folder is not a package):

```
commit_sha <- packageDescription("VDCNNN")$RemoteSha
branch_name <- packageDescription("VDCNNN")$RemoteRef
```